ios/RNFishjamClient.swift

```diff
@@ -243,39 +244,107 @@ class MembraneWebRTC: MembraneRTCDelegate {
                     "No endpoints available. Ensure the connection is established or endpoints are present.")
         }
     }
-
-    func receiveMediaEvent(data: String) throws {
-        try ensureConnected()
-        membraneRTC?.receiveMediaEvent(mediaEvent: data as SerializedMediaEvent)
+  
+  func onSocketClose(code: UInt16, reason: String) {
```

Comment:
I think we still do not have `swift-lint` configured for this repo. But in meantime I think you can fix indentations here to match rest of file